### PR TITLE
Blog-app: Processing

### PR DESCRIPTION
**Pull request by Ceemos96**
### In this milestone, I:
- Added the desired data inside pgadmin with rails console
- Added the desired CRUD models on rails repo with rails console

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```
Comment (fobadara) [inline app/models/user.rb]: [REQUIRED] Foreign keys should be `author_id`, not `user_id`

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'users_id'
```
Comment (fobadara) [inline app/models/post.rb]: [REQUIRED] Kindly remove the foreign_key since it is not needed here

File: app/models/comment.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'users_id'
```
Comment (fobadara) [inline app/models/comment.rb]: [REQUIRED] Kindly remove the. foreign_key since it is not needed here

File: app/models/like.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'users_id'
```
Comment (fobadara) [inline app/models/like.rb]: [REQUIRED] Kindly remove the foreign_key since it is not needed here

File: app/models/comment.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+  belongs_to :post
+
+  def update_comments_counter
```
Comment (fobadara) [inline app/models/comment.rb:7]: [OPTIONAL] It wouldbeniceif this is run ater_save just like in post

File: app/models/like.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:Likes_Counter)
+  end
```
Comment (fobadara) [inline app/models/like.rb:9]: [OPTIONAL] It would be nice if this is run after_save  just like in post

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Ceemos96, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @Ceemos96, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job processing data models correctly 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**